packages/aws_common/test/http/http_common.dart

```diff
@@ -56,8 +56,10 @@ void clientTest(
           client = debugClient..supportedProtocols = supportedProtocols;
         });
         tearDown(() async {
-          httpServerChannel.sink.add(null);
-          // await Future<void>.delayed(const Duration(seconds: 30));
+          httpServerChannel.sink.rejectErrors().add(null);
+        });
+        tearDownAll(() async {
+          // await Future<void>.delayed(const Duration(seconds: 60));
```

Comment:
Is the `tearDownAll` needed?

Reply:
It's the best way to debug browser tests, since the browser console logs does not output to the Dart console. I end up needing it every time I update HTTP code so I'd rather keep it 😅

Reply:
lol OK, I used the same thing making tests not to finish.